Core 2.0.0 Release Candidate.

ravi-prakash-v opened the pull request:
Creating baseline version for 2.x Beckn Protocol specification.

anandp504 committed:
Add mediasearch capability

Commit:
Added CC-BY-NC-SA 4.0 license

anandp504 committed:
Change folder structure and modify jsonld

anandp504 committed:
Provide beckn prefix definition

anandp504 committed:
Move core schemas under core/v2

anandp504 committed:
Remove schema-packs directory

anandp504 committed:
Add documentation for schema bundles

anandp504 committed:
Add migrations, validations folders to README.md

anandp504 committed:
Reuse ack responses

anandp504 committed:
Fix profile.json and renderer.json

anandp504 committed:
Move offer apis to discover.yaml

anandp504 committed:
Merge pull request #19 from beckn/api-changes

Change folder structure and rearrange core schemas

anandp504 committed:
Core offer schema changes

anandp504 committed:
Modify core offer object

anandp504 committed:
Change value to price in priceSpecification

anandp504 committed:
Remove enum for payment methods

anandp504 committed:
Modify vocab.jsonld for the core offer changes

anandp504 committed:
Order and Item avaiabilityWindow changes

anandp504 committed:
Core Payment changes

anandp504 committed:
Add form to core spec for ratings

anandp504 committed:
Keep attributes.yaml and vocab.jsonld in sync

anandp504 committed:
Remove payment method from Payment object

anandp504 committed:
Fix enums for payment methods

anandp504 committed:
Merge pull request #20 from beckn/offer-schema-changes

Core offer schema changes

anandp504 committed:
Add charging telemetry for EV Charging Session

anandp504 committed:
Add message envelope to catalog publish apis

anandp504 committed:
Add core context.jsonld and vocab.jsonld

anandp504 committed:
Add Eligibility object to the core.yaml

anandp504 committed:
EvCharging offer, session, service as separate folders

anandp504 committed:
Merge pull request #21 from beckn/schema-reorg

EvCharging offer, session, service as separate folders

anandp504 committed:
Merge protocol api specs

anandp504 committed:
Remove unreference Party object

anandp504 committed:
Move acceptedPaymentMethod to core Item

anandp504 committed:
Fix context URI for core entities

anandp504 committed:
Allow spatial filters in discover requests

anandp504 committed:
Fix ChargingSession schema and core schema

anandp504 committed:
Merge pull request #22 from beckn/ev-order-changes

Fix ChargingSession schema and core schema

anandp504 committed:
Fix core objects and context uri for EvCharging objects

anandp504 committed:
Add EvCharging enums to jsonld construct

anandp504 committed:
Add core enums to jsonld construct

anandp504 committed:
Add isActive flags to catalog and offer

anandp504 committed:
Use Enumeration instead of DefinedTerm

anandp504 committed:
Modify quantity object

anandp504 committed:
Deleted individual api spec files

anandp504 committed:
Fix amenityFeature enum values

anandp504 committed:
Merge pull request #23 from beckn/enum-changes

Core and EvCharging Enum changes

anandp504 committed:
Fix Rating object required properties

anandp504 committed:
Fix payment object required attributes

ravi-prakash-v committed:
Add initial governance documentation

ravi-prakash-v committed:
Add initial contents section to README

ravi-prakash-v committed:
Create CONTRIBUTING.md with contributing guidelines

Add initial contributing guidelines for the specification

ravi-prakash-v committed:
Create README.md

anandp504 committed:
Fix the buyer field in Order object

anandp504 committed:
Update track and cancel api specs

anandp504 committed:
Restructure folders and use main branch for references

ravi-prakash-v committed:
Merge pull request #32 from beckn/folder-restructuring

Restructure folders and use main branch for references

ravi-prakash-v committed:
Merge pull request #31 from beckn/draft

Beckn protocol v2 schema and api changes

anandp504 committed:
Introduce bppId and bppUri in Catalog object

anandp504 committed:
Make bppId, bppUri mandatory for Catalog

ravi-prakash-v committed:
Merge pull request #33 from beckn/bppId-changes

Include bppId and bppUri in the core Catalog object

ravi-prakash-v committed:
Merge pull request #34 from beckn/draft

Draft

ravi-prakash-v committed:
Add Beckn Protocol v2.0.0-beta specification

This commit introduces the pre-release specification for Beckn Protocol v2.0.0-beta, detailing its core schema, updated APIs, and key differences from v1.x. It emphasizes the new Catalog Discovery Service (CDS) and the DeDi-compliant Network Registry, along with design-time and run-time composability benefits.

anandp504 reviewed (approved):
Approved 👍

pramodkvarma reviewed (approved):
Please see my comments, fix those, and merge.

ravi-prakash-v committed:
Revise README for Beckn Protocol v2.0.0-beta

Updated README.md to reflect changes in Beckn Protocol v2.0.0-beta, including key differences, schema changes, and benefits.

abhimail reviewed (approved):
Approved

ravi-prakash-v commented on README.md:
```diff
@@ -0,0 +1,314 @@
+# Beckn Protocol v2.0.0 (beta release)
```
Noted.
